src/server.js

```diff
@@ -6,7 +6,7 @@ const mongoose = require('mongoose');
 const dotenv = require('dotenv');
 const morgan = require('morgan');
 const cors = require('cors');// opens up our server for any domain to access
-const { response } = require('express');
+const Users = require('./auth/models/user-schema.js');
 
 const basicAuth = require('./auth/middleware/basic-auth.js');
```

Comment:
change this to `const authRouter = require('./routes/router');` and change line 30 to `authRouter` because it looking for a route when `/signup` or `/signin` is hit. Then in router.js it will go to the basic-auth